src/uct/sm/base/sm_iface.c

```diff
@@ -15,6 +15,18 @@
 #include <ucs/sys/string.h>
 #include <ucs/sys/sys.h>
 #include <ucs/arch/cpu.h>
+#include <ucs/type/init_once.h>
+
+
+typedef struct {
+    uint64_t ext:1; /* extended address is used */
+    uint64_t id :63;
```

Comment:
better use bitmap because bitmap is not portable and this struct is passed to remote side